backend/python/app/backend/.env.example

```
@@ -0,0 +1,3 @@
+DEBUG=True
+SECRET_KEY=your-secret-key
+ALLOWED_HOSTS=*
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ALLOWED_HOSTS key should go before the DEBUG key

(UnorderedKey)